src/functionaltests/java/com/ericsson/ei/utils/FunctionalTestBase.java

```diff
@@ -46,10 +53,22 @@
     @Autowired
     private MongoProperties mongoProperties;
 
+    @Value("${spring.data.mongodb.database}")
+    private String database;
+
+    @Value("${event_object_map.collection.name}")
+    private String collection;
+
+    private MongoClient mongoClient;
+    
     public int getMongoDbPort() {
         return mongoProperties.getPort();
     }
-
+    
+    public String getMongoDbHost() {
+        return mongoProperties.getHost();
+    }
+    
     @Override
     public void beforeTestClass(TestContext testContext) throws Exception {
         int debug = 1;
```

Comment:
Where and What is this debug int values used?

Reply:
Do not know.